anitya/tests/test_flask.py

```diff
@@ -683,7 +683,7 @@ def test_projects_search_by_name(self):
         create_project(self.session)
 
         output = self.app.get("/project/geany")
-        self.assertEqual(output.status_code, 301)
+        self.assertEqual(output.status_code, 308)
```

Comment:
This seems to be a change in a recent version of either flask or werkzeug, I wonder if you shouldn't do something like:
``self.assertTrue(output.status_code in [301, 308])``

Reply:
I updated this to `assertIn`

I found some documentation about the error 308 and it should be more reliable than 301 (doesn't allow change from `POST` to `GET`). See https://httpstatuses.com/308

Reply:
I've just checked, it's indeed a change in werkzeug: https://werkzeug.palletsprojects.com/en/0.15.x/changes/#version-0-15-0 (4th bullet point)